### DISCUSSION TITLE
mounted!! Walk two times!!

### DISCUSSION THREAD

**Discussion by 983796605** [nuxt/nuxt#9597] [Questions (Nuxt 2)] [answered]
What's the reason for going twice

**Comment by kissu** [comment, marked as answer]
Hey, could you please give us some more context and some code? Pretty hard to help here.

**Reply by 983796605** [reply]
The client-side rendered virtual DOM tree is not matching server-rendered content. This is likely caused by incorrect HTML markup, for example nesting block-level elements inside <p>, or missing <tbody>. Bailing hydration and performing full client-side render.
Does this error cause Mounted to execute twice

**Reply by kissu** [reply]
Check this one: https://stackoverflow.com/questions/47862591/vuejs-error-the-client-side-rendered-virtual-dom-tree-is-not-matching-server-re

**Reply by 983796605** [reply]
Have you ever seen life cycle Mounted execute twice

**Reply by kissu** [reply]
Yep, probably happening because of the DOM mismatch. You app renders something, then it renders it a second time because it failed the first time. Hence, twice the lifecycle.

**Reply by 983796605** [reply]
Excuse me, how to solve this? After login, the API in Mounted is executed twice.
Our website is: www.acc5.com

**Reply by kissu** [reply]
I've given some decent starting point in the stackoverflow link.
Also, some code could help us debug all of this.

**Reply by 983796605** [reply]
Our production environment added CDN caching.
A double execution problem occurs.
Our test environment has no CDN cache.
There is no double execution problem.
Could it be CND cache?

**Reply by kissu** [reply]
Maybe, try to look what is the content generated on the server and the one hydrated on the client to spot any differences.

**Reply by 983796605** [reply]
thanks
